### ISSUE TITLE
Models

### ISSUE THREAD

**Issue by zenrabbit** [MarioZuliani/Chapter-5#1]
## Main models

y = n_animals, n_species, evenness

models

m = glm(y ~ microsite * site + temp, family = quasipoisson or gaussian, data = data)

### Secondary models (to test if mimics changed temp for instance)
m = glm(temp ~ microsite * site, .....)?

**Comment by zenrabbit** [comment]
### Main stats in Chapter 5.rmd
line 284   
anova_result <- aov(mean_pendant_temp ~ factor, data = pendant_temp) ### Temp is significant.  

just add site in, what is factor?

line 331 onwards are main models  
see above

**Comment by zenrabbit** [comment]
yes, check distribution and then check overdispersion of models too
https://easystats.github.io/performance/reference/check_overdispersion.html#:~:text=If%20the%20dispersion%20ratio%20is,05%20indicates%20overdispersion.

**Comment by zenrabbit** [comment]
if site not s.d. in temp or any of the y measures, drop?

**Comment by zenrabbit** [comment]
@MarioZuliani pitch your results here simply and see if Nargol will review to ensure smooth flow.
map
boxplots
species frees

then rest in supplement

and covariates simply stated after main findings

mimic benefit decreases with increasing local shrub densities?
mimics cooled

@nargol about air temp vs ground? I assume air temp better for animal studies? are both temp measures correlated?

OK checking out now.

**Comment by MarioZuliani** [comment]
Map options generated at end of Html.
Agree with figure layout (all will be combined into one figure).
Here is a simplified findings. Let me know what you think. We have a TON of findings to report.

Findings simplified:
**Microsite**
Abundance significantly affected by Microsite (Mimic > Open, Shrub > Open, Mimic = Shrub)
Richness significantly affecter by microsite (Mimic > Open)
Evenness Not significantly affected by microsite.

**Ambient Temp**
Abundance not significant.
Richness microsite*temp significant with increasing temp(mimic > open, mimic > shrub)
Evenness not significant.

**Ground Temp**
Abundance microsite*ground_temp significant  with increasing temp(Mimic > Open, Mimic > Shrub, Open > Shrub)
Richness not significant.
Evenness Not significant.

**Shrub Density**
Abundance significantly negative impact by density but not microstie*density
Richness significantly negative impact by density but not microstie*density
Evenness not significant.

**Community Composition**
Community composition did not significantly vary across microsites but DID across shrub densities.

**Comment by zenrabbit** [comment]
- map, change point color, zoom out, get rid of 'color' legend on size, zoom out if just 4 points
- can you add a map with all the plots? see what that looks like?

**Comment by zenrabbit** [comment]
recommend this color palette 
+
  scale_fill_brewer(palette = "Paired")
![Unknown](https://github.com/MarioZuliani/Chapter-5/assets/4825462/d917297f-ad69-4601-9105-28ce70b253e6)

**Comment by nargolg1** [comment]
Hi Team, 

I agree with Chris, run temperature models and test which sites and mimics differed using emmeans (which I think you already have for microsites but not sites maybe).
Last thing is RII, but I would say submit without and then when the reviews come back you might have to do it. 
How did you compare ground temperature and richness and abundance, weren't they handhelds? I think you don't have enough reps to be able to compare them accurately. You can compare ground vs ambient temp however and just use ambient (Near-surface air temperature) in models instead.

**Comment by zenrabbit** [comment]
Near-Surface Air Temperatures are the best. MANY papers
@nargolg1 if you have a good citation for Mario, can you pls provide? I cannot find mine. I had a good one.

**Comment by zenrabbit** [comment]
got it, 
Foundation plant species provide
resilience and microclimatic
heterogeneity in drylands

https://www.nature.com/articles/s41598-022-22579-1

citation #35 in paper, plus cite the paper too
Terando, A., Youngsteadt, E., Meineke, E. & Prado, S. Accurate near surface air temperature measurements are necessary to gauge large-scale ecological responses to global climate change. Ecol. Evol. 8, 5233–5234. https://doi.org/10.1002/ece3.3972 (2018).
